tests/kernel/install_ltp.pm

```diff
@@ -179,16 +179,18 @@ sub upload_runtest_files {
     my $aiurl = autoinst_url();
 
     my $up_script = qq%
-rfiles=\$(ls --file-type $dir)
-for f in \$rfiles; do
-    echo "Uploading ltp-\$f-$tag"
-    curl --form upload=\@$dir/\$f --form target=assets_public $aiurl/upload_asset/ltp-\$f-$tag
-    echo "ltp-\$f-$tag" >> /tmp/ltp-runtest-files-$tag
-done
-curl --form upload=\@/tmp/ltp-runtest-files-$tag --form target=assets_public $aiurl/upload_asset/ltp-runtest-files-$tag
-curl --form upload=\@/root/openposix-test-list-$tag --form target=assets_public $aiurl/upload_asset/openposix-test-list-$tag
+ldir='/tmp/runtest-files-$tag'
+archive="\$ldir.tar.gz"
+mkdir -p \$ldir
+cd \$ldir
+ls --file-type $dir | sed "s/\\(.*\\)/ltp-\\1-$tag/" > \$ldir/ltp-runtest-files-$tag
```

Comment:
Do we still need to prepend 'ltp-' if the files are stored in an archive?

Reply:
No, I just removed it.

Reply:
The only files with prepend 'ltp-' are these 2 lists (opensuse and LTP): openposix-test-list-opensuse-Tumbleweed-x86_64-20200307-DVD@64bit-with-ltp-qcow2 and ltp-runtest-files-opensuse-Tumbleweed-x86_64-20200307-DVD@64bit-with-ltp-qcow2. Note I also copied openposix, which is not needed (but that's not a problem).